distribution/test/itests/test-itests-common/pom.xml

```diff
@@ -103,6 +103,17 @@
             <artifactId>platform-util</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.xebialabs.restito</groupId>
```

Comment:
The comment at the top of that class actually mentions that it's a workaround for a commit that @ryeats contributed to restito, which should be in version 0.8.2. Maybe we should just upgrade to that version instead? I'll make a ticket to address updating our restito version to 0.8.2 and removing the ddf.test.thirdparty:Restito class. It would also probably be best if we had a restito variable since it's version is declared specifically in a few places.